pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -121,7 +121,9 @@ public void run() {
 
     @Option(
         names = "--format",
-        description = "The type of data to be imported (default: ${DEFAULT-VALUE}).",
+        hidden = true,
+        description =
+            "The type of data to be imported, possible values are: ${COMPLETION-CANDIDATES} (default: ${DEFAULT-VALUE}).",
```

Comment:
I didn't know `${COMPLETION-CANDIDATES}` existed.  Looks like I should open another PR to add this to other options as well.